app/assets/javascripts/summernote-init.js

```diff
@@ -5,7 +5,8 @@ $(document).on('ready', function() {
       toolbar: [
         ["style", ["bold", "italic", "underline", "clear"]],
         ["table", ["table"]],
-        ["para", ["ul", "ol", "paragraph"]]
+        ["para", ["ul", "ol", "paragraph"]],
+        ['insert', ['link']]
```

Comment:
could you use double quotes here to match the rest of the list?

Reply:
oops